test/library/packages/Image/example1.chpl

```diff
@@ -0,0 +1,16 @@
+use Image;
```

Comment:
this changes the docs in that `use Image` is no longer in the example (but it was in your original RST). just pointing that out in case it's not deliberate.

Reply:
That was deliberate. I felt like that was stating the obvious